src/app/application.cpp

```diff
@@ -403,15 +403,63 @@ void Application::processParams(const QStringList &params)
         return;
     }
 #endif
+    BitTorrent::AddTorrentParams torrentParams;
+    bool skipTorrentDialog = false;
 
     foreach (QString param, params) {
         param = param.trimmed();
+
+        // Process strings indicating options specified by the user. Use of
+        // any of these should prevent the "Add new torrent" dialog from
+        // opening:
+
+        if (param.startsWith("@path=")) {
+            torrentParams.savePath = param.mid(6);
+            skipTorrentDialog = true;
```

Comment:
Because utilizing any of the command line options relating to adding a torrent prevents the "Add new torrent" dialog from opening.